### ISSUE TITLE
🚧Update github solution folders and items

### ISSUE THREAD

**Issue by CalvinWilkinson** [KinsonDigital/CASL#69]
### I have done the items below . . .

- [X] I have updated the title by replacing the '**_<title_**>' section.

### Description

Add all of the various files and folders in the _**.gihub**_ folder to the solution as solution folders and item references.  This is to make it easy to see and open the various folders and files in the solution.   This applies to both **Visual Studio** and **Rider**.

Currently is this:
![image](https://user-images.githubusercontent.com/85414302/133982736-7d9de02a-7e34-4e03-8b8d-85b566723928.png)

Should be like this:
![image](https://user-images.githubusercontent.com/85414302/133982786-317d90fa-226e-477e-93f8-02401c5f1591.png)

### Acceptance Criteria

**This issue is finished when:**
- [ ] Various folders and files in the _**.github**_ folder added as solution items

### ToDo Items

- [ ] Draft pull request created and linked to this issue
- [X] Priority label added to issue (low priority, medium priority, or high priority)
- [X] Issue linked to the proper project
- [X] Issue linked to proper milestone
- [X] Unit tests have been written and/or adjusted for code additions or changes
- [X] All unit tests pass

**Comment by teezzan** [comment]
Can I hop on this? Against tomorrow?

**Comment by CalvinWilkinson** [comment]
Yes, go ahead.  When you create the pull request, this project is in the same situation currently as Velaptor. 

Create the branch off of the _**release/v1.0.0**_ branch and the PR should merge it back into _**release/v1.0.0**_.

Thanks!!

**Comment by teezzan** [comment]
Just to clarify. I need to create a `SolutionItems` folder and copy the content of  `.github` into it? If that is correct, can you explain this?
>This is to make it easy to see and open the various folders and files in the solution. This applies to both Visual Studio and Rider.

I am trying to learn while making the little contribution that I can. I appreciate the links to the resources that you always reference in your messages. Thank you.

**Comment by CalvinWilkinson** [comment]
I apologize if you already know what the difference is between a solution folder in **Visual Studio** or **JetBrains Rider** and an actual folder on disk, but just in case you do not, I will explain it.

A solution folder is nothing but a **virtual** folder (not on disk), that is used by both IDEs to further organize things in your solution or project.  A file that is added to a solution folder is nothing but a **pointer / reference** to a real file on disk, and that **pointer / reference** is saved in the _**.sln**_ file. 

If you create a solution folder in any C# project using these IDEs, you will notice the GIT changes in your git client.

Basically I want to give the user/developer to easily open the files in the IDE, but they cannot do that unless they open it in some other editor OR just manually open it in the same IDE that they are in.  This is merely a convenience that we are introducing here.

If you want to see an example that is already setup, pull down the [Velaptor](https://github.com/KinsonDigital/Velaptor) project, checkout the _**v1.0.0-preview.3**_ branch, and open it in either IDE and you will see what I am going for.  Just copy what Velaptor has going on and your good to go.

I hope this clears things up!!  Let me know if you have anything else.
